### PR TITLE
Support for building ngraph EP on Windows

### PR DESCRIPTION
**Pull request by rgesteve**
Builds on PR #940 (merged in 3b0dda0aca).

### CODE REVIEW

**Comment by jywu-msft** [comment]
Thanks for the PR. I set up the win nGraph CI pipeline. Looks like there's a build failure. Please take a look.

**Comment by rgesteve** [comment]
Thank you, @jywu-msft.  Is there a way I can kick off the CI to test it?  I basically copied the windows version of the azure pipelines, but knew of no way of testing whether it was correct.

File: cmake/external/ngraph.cmake [hunk 1]
```diff
@@ -14,45 +14,95 @@ set(ngraph_URL "https://github.com/NervanaSystems/ngraph.git")
...
-set_property(TARGET ngraph PROPERTY IMPORTED_LOCATION ${ngraph_LIBRARIES}/${NGRAPH_SHARED_LIB})
+# discard prior changes due to unblock incremental builds.
+set(NGRAPH_PATCH_DISCARD_COMMAND cd ${ngraph_SRC} && git checkout -- .)
+
+if (MSVC)
+    set(prebuilt_ONNX_BINARY_DIR "${CMAKE_CURRENT_BINARY_DIR}/onnx/${CMAKE_BUILD_TYPE}")
+    set(prebuilt_ONNX_SOURCE_DIR "${CMAKE_CURRENT_BINARY_DIR}")
+
+
+    # For the moment, Windows does not support codegen, it works on DEX-only mode
```
Comment (jywu-msft) [inline cmake/external/ngraph.cmake]: when should we expect codegen support?

Comment (jfdong) [reply cmake/external/ngraph.cmake]: @jywu-msft  May I know what is the special value that codegen has but direct execution (DEX mode) can't support.

**Pull request merged**